epochStart/shardchain/trigger.go

```diff
@@ -586,7 +590,7 @@ func (t *trigger) receivedMetaBlock(headerHandler data.HeaderHandler, metaBlockH
 		t.newEpochHdrReceived = true
 		t.mapEpochStartHdrs[string(metaBlockHash)] = metaHdr
 		// waiting for late broadcast of mini blocks and transactions to be done and received
-		wait := common.ExtraDelayForRequestBlockInfo
```

Comment:
remove the constant?

Reply:
this const is used also in `baseProcessor.go` file